contracts/contracts/strategies/MorphoCompoundStrategy.sol

```diff
@@ -190,24 +190,27 @@ contract MorphoCompoundStrategy is BaseCompoundStrategy {
         address _asset,
         uint256 _amount
     ) external override onlyVault nonReentrant {
-        _withdraw(_recipient, _asset, _amount);
+        _withdraw(_recipient, _asset, _amount, false);
     }
 
     function _withdraw(
         address _recipient,
         address _asset,
-        uint256 _amount
+        uint256 _amount,
+        bool _skipAmountCheck
     ) internal {
-        require(_amount > 0, "Must withdraw something");
+        require(_amount > 0 || _skipAmountCheck, "Must withdraw something");
         require(_recipient != address(0), "Must specify recipient");
 
         address pToken = assetToPToken[_asset];
         uint256 oraclePrice = ORACLE.getUnderlyingPrice(pToken);
 
-        IMorpho(MORPHO).withdraw(pToken, _amount.divPrecisely(oraclePrice));
+        if (_amount > 0) {
```

Comment:
`if (_amount > 0) {` is a duplicate of the stronger `require(_amount > 0)` above it.

Reply:
Now that the `skipAmount` param is removed this line plays no role anymore... removed